File-Formats/PDF/Convert-HTML-To-PDF/blazor.md

```diff
@@ -11,6 +11,8 @@ keywords: Assemblies
 
 The Syncfusion HTML to PDF converter is a .NET library used to convert HTML or web pages to PDF document in Blazor application.
 
+N> Our HTML to PDF converter is only supported in Blazor Server-Side, currently it is not supported in Blazor WASM.
```

Comment:
Use this "At present, our HTML to PDF converter is exclusively supported in Blazor Server-Side, while it is not compatible with Blazor WASM (WebAssembly)."

Reply:
Modified the sentence